conf/httpd.conf

```diff
@@ -50,7 +50,18 @@ TypesConfig /etc/mime.types
 DefaultType text/plain
 MIMEMagicFile conf/magic
 HostnameLookups Off
-ErrorLog /dev/stderr
+<IfDefine NETCAT_LOGS>
+    ErrorLog "|/usr/bin/nc localhost 5880"
+    <IfDefine ACCESS_LOGS>
+        TransferLog "|/usr/bin/nc localhost 5880"
```

Comment:
these should probably be `${LOGGING_PORT}` instead of `5800`.  (i think this will work; untested).